vendor/validator.js

```diff
@@ -79,329 +85,356 @@
       success: 'glyphicon-ok',
       error: 'glyphicon-remove'
     }
-  }
+  };
 
   Validator.VALIDATORS = {
-    'native': function ($el) {
-      var el = $el[0]
+    'native': function($el) {
+      var el = $el[0];
+
       if (el.checkValidity) {
-        return !el.checkValidity() && !el.validity.valid && (el.validationMessage || "error!")
+        return !el.checkValidity() && !el.validity.valid && (el.validationMessage || 'error!');
       }
     },
-    'match': function ($el) {
-      var target = $el.attr('data-match')
-      return $el.val() !== $(target).val() && Validator.DEFAULTS.errors.match
+    'match': function($el) {
+      var target = $el.attr('data-match');
+
+      return $el.val() !== $(target).val() && Validator.DEFAULTS.errors.match;
     },
-    'nomatch': function ($el) {
-      var target = $el.attr('data-nomatch')
-      return $el.val() === $(target).val() && Validator.DEFAULTS.errors.nomatch
+    'nomatch': function($el) {
+      var target = $el.attr('data-nomatch');
+
+      return $el.val() === $(target).val() && Validator.DEFAULTS.errors.nomatch;
     },
-    'minlength': function ($el) {
-      var minlength = $el.attr('data-minlength')
-      return $el.val().length < minlength && Validator.DEFAULTS.errors.minlength
+    'minlength': function($el) {
+      var minlength = $el.attr('data-minlength');
+
+      return $el.val().length < minlength && Validator.DEFAULTS.errors.minlength;
+    },
+    'url-contains': function($el) {
```

Comment:
It looks like you're updating the third-party vendor library in order to implement new validators.

http://1000hz.github.io/bootstrap-validator/#validator-options suggests that custom validators can be implemented without changing the library. Please do it without changing the library if possible.

This way, you also don't need to apply linting to the third-party vendor library since this PR is extremely difficult and risky to review in the current state.